cypress/e2e/candidate.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Candidate Interview Result Verification Pass/Fail
+    Scenario: The Admin should successfully transition the candidate's status to  Interview Passed
+        Given  Admin login
+        Given Created Employee
+        Given Created job
+        Given Created Vacancy
+        Given Created Candidate
+        When Change the candidate status to Interview Passed
+        Then Successfully transition the candidate's status to Interview Passed
+    Scenario: The Admin should successfully transition the candidate's status to Interview Failed
+        Given  Admin login
+        Given Created Employee
+        Given Created job
+        Given Created Vacancy
+        Given Created Candidate
```

Comment:
When put these in before hook we reduce redundant code and don't declare them one more time in this scenario cuz these data same for scenario 1 and 2 so shared data preparation in before hook